frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -1001,6 +1002,26 @@ export const ClusterVersionDetailsTable: React.FC<ClusterVersionDetailsTableProp
             <dd>
               <ResourceLink kind={referenceForModel(ClusterVersionModel)} name={cv.metadata.name} />
             </dd>
+            <dt>{t('public~Upstream configuration')}</dt>
+            <dd>
+              <div>
+                {!cv.spec.upstream ? (
+                  <span className="text-muted">{t('public~Default configuration: ')}</span>
```

Comment:
I wonder if we should display just "Default update server" and not put the URL. As a user, I don't think I care what the default URL actually is. I just care that it's configured to use the default.